expenses-manager-frontend/src/main/frontend/src/app/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,43 @@
+.sidebar{
+    width: 220px;
+    height: 100%;
+    background-color: #4F46E5;
+
+    .content{
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        @mixin font{
+            font-family: 'Outfit';
+            font-size:large;
+            color: #E9F0FF;
+        }
+        @include font;
+        
+        .logo{
+            margin: 5px;
+        }
+
+        .sidebar-button{
+            margin: 5px;
+            width: 90%;
+            background-color: #4F46E5;
+            @include font;
+            font-size:medium;
```

Comment:
Valorile font-size: medium;, precum și alte dimensiuni relative în CSS, pot varia în funcție de configurațiile de dispozitive, browsere și preferințele utilizatorilor. As sugera sa fim constanti in definirea dimensiunilor si sa folosim doar rem.